app/src/organisms/InterventionModal/StackerFillInterventionContent.tsx

```diff
@@ -0,0 +1,115 @@
+import { ComponentProps } from 'react'
```

Comment:
```suggestion
import type { ComponentProps } from 'react'
```
and move down to the type import section